jujugui/static/gui/src/app/app.js

```diff
@@ -686,20 +686,20 @@ YUI.add('juju-gui', function(Y) {
       var msg = (
         <p>
           Find your username and password with<br />
-          <code>`juju show-controller --show-passwords`</code>
+          <code>juju show-controller --show-passwords</code>
         </p>);
       if (this.get('jujuCoreVersion').lastIndexOf('1.', 0) === 0) {
         // Use old command to retrieve the password.
         msg = (
           <p>
             Find your password with<br />
-            <code>`juju api-info --password password`</code>
+            <code>juju api-info --password password</code>
           </p>);
       }
       document.getElementById('loading-message').style.display = 'none';
       ReactDOM.render(
         <window.juju.components.Login
-          envName={this.db.environment.get('name') || 'Sandbox'}
+          envName={this.get('jujuEnvUUID') || this.db.environment.get('name')}
```

Comment:
I'm a little concerned that this might actually get a 'uuid' instead of model name. Something to check in qa.